default.nix

```diff
@@ -119,10 +119,13 @@ rec {
       '';
       doCheck = true;
       checkPhase = ''
-        test -n "$(find $out/haskell/ -type f -name '*.hs')"
+        test -n "$(find $out/ -type f -name '*.hs')"
         # OUT_DIR=$out make "${project}".hsTest
       '';
-      dontInstall = true;
+      installPhase = ''
+        mv $out/haskell/${main}/* $out
+        rm -rf $out/haskell
+      '';
```

Comment:
NB: we now have to be careful not to generate both Ledger and MidnightExample under the same directory.

Reply:
I'm not sure what you mean by that, Nix has different outputs for midnight and ledger, so that shouldn't ever happen

Reply:
c.f. `build.yml`, where we call `nix-build` twice (once for Ledger and once for MidnightExample` but then `rsync` their output folders under a common `docs/` folder.

Reply:
oh, yeah, that's going to cause problems